### PR TITLE
Fixed code linting issues.

### PR DESCRIPTION
**Pull request by sonnykt**
JIRA: https://govdex.gov.au/jira/browse/GCMSLG-19

Next step:
Update the PHP Mess Detector command (`phpmd`) in `/usr/bin/lint-govcms` (on govCMS Docker CLI image) to:
* not include features as we cannot control files generated by the Features module.
* include the option `--ignore-violations-on-exit` so that `phpmd` exits with a zero code even if any violations are found. Reports from phpmd should be considered as suggestions only. Most of reports from phpmd regard to unused variables from function signature of hook implementations.

### CODE REVIEW

File: govcms.install [hunk 1]
```diff
@@ -6,6 +6,8 @@
  *
  * @license GPL v2 http://www.fsf.org/licensing/licenses/gpl.html
  *
+ * @codingStandardsIgnoreStart
```
Comment (sonnykt) [inline govcms.install:9]: Ignore the section below to keep the copyright text (for legal purpose?).

File: themes/govcms/govcms_barton/govcms_barton.info [hunk 2]
```diff
@@ -1,3 +1,4 @@
+; @codingStandardsIgnoreFile
```
Comment (sonnykt) [inline themes/govcms/govcms_barton/govcms_barton.info:1]: The info file has some example PHP code in its comment, and CodeSniffer picks them up and treats the file as PHP.

File: phpcs.xml [hunk 3]
```diff
@@ -11,7 +11,6 @@
   <arg value="sp"/>
 
   <file>/app/profiles/govcms/modules/custom</file>
-  <file>/app/profiles/govcms/modules/features</file>
```
Comment (sonnykt) [inline phpcs.xml:14]: All features should be excluded from CodeSniffer.

**Comment by tobybellwood** [comment]
Can we update the phpmd to remove the features and ignore-violations as part of this PR? 

```
# Check code mess.
${APP_DIR}/tests/vendor/bin/phpmd ${PROFILE_DIR}/modules/custom,${PROFILE_DIR}/themes/govcms,tests/behat/bootstrap,${PROFILE_DIR}/govcms.info,${PROFILE_DIR}/govcms.install,${PROFILE_DIR}/govcms.profile text codesize,unusedcode,cleancode --ignore-violations-on-exit
```

**Review by gargsuchi — APPROVED** [review]

**Comment by sonnykt** [comment]
> Can we update the phpmd to remove the features and ignore-violations as part of this PR?
> 
> ```
> # Check code mess.
> ${APP_DIR}/tests/vendor/bin/phpmd ${PROFILE_DIR}/modules/custom,${PROFILE_DIR}/themes/govcms,tests/behat/bootstrap,${PROFILE_DIR}/govcms.info,${PROFILE_DIR}/govcms.install,${PROFILE_DIR}/govcms.profile text codesize,unusedcode,cleancode --ignore-violations-on-exit
> ```

Hi @tobybellwood, I've added another commit to update phpmd.

**Pull request merged**